demo/app/lib/views/wallet_initiated_connect.dart

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import 'dart:convert';
+import 'dart:developer';
+import 'package:app/views/supported_credentials_list.dart';
+import 'package:app/widgets/primary_button.dart';
+import 'package:flutter/material.dart';
+import 'package:app/widgets/common_title_appbar.dart';
+import 'package:flutter/services.dart';
+import 'package:app/wallet_sdk/wallet_sdk_mobile.dart';
+
+class IssuerList extends StatefulWidget {
+  const IssuerList({Key? key}) : super(key: key);
+
+  @override
+  State<IssuerList> createState() => IssuerListState();
+}
+
+class IssuerListState extends State<IssuerList> {
+  Map<String, String> issuerMetadata = {};
+  Map<String, dynamic> values = {};
+  var WalletSDKPlugin = WalletSDK();
+
+  readIssuerAuthFlowConfig() async {
+    final String response = await rootBundle.loadString('lib/assets/walletConnectIssuerConfig.json');
```

Comment:
It is a bad idea to work with JSON from config directly on the View component. It is hard to understand what going on, and if in the future we changed the structure of this JSON file, the code of view needs to be changed. Parse JSON first into some structure like ConnectIssuerConfig, and then use it.

Reply:
https://github.com/trustbloc/wallet-sdk/blob/83f23ca264b71d08dc06d734578a8897f84ec5d8/demo/app/lib/models/connect_issuer_config.dart

Have addressed your comment